### ISSUE TITLE
[Accessibility] - Need help Signing In button not actionable with NVDA screen reader

### ISSUE THREAD

**Issue by ramjiseetharaman** [okta/okta-signin-widget#1890]
:information_source: **If you have a question**, please post it on the [Okta Developer Forum](https://devforum.okta.com) instead. Issues in this repository are reserved for bug reports and feature requests.

### I'm submitting a
  - [X ] bug report
  - [ ] feature request

### Background info
Need Help Signing In link is not actionable when NVDA screen reader is turned on.

### Expected behavior

Links needs to be properly structured. In this case, since the action expands/collapses the section with a click handler, recommendation is to make this a native button and style it like a link. This way native keyboard accessibility is provided for the buttons. Don't use ::after to render the link which is not accessible to most browser assistive technology combination.

### What went wrong?

Whenever you turn on NVDA screen reader and try to activate the "Need help Signing In" link, it does not expand. Not able to interact with the link when NVDA screen reader is turned on. The link is not properly formed and it uses the ::after pseudo class to provide the content and used to render the link.

### Steps to reproduce

1. Navigate to the OKTA Sign In page
2. Download the NVDA Screen reader (https://www.nvaccess.org/download/)
3. Turn on NVDA screen reader
4. Navigate to the "Need Help Signing In" link in the page using Tab keystroke
5. Activate the link using Enter/Spacebar key
6. Verify whether the section expands/collapses when NVDA is turned on. (Issue)

### Your environment

  - Okta Sign-In Widget Version: ^3.8.1
  - Browser: Chrome
  - OS: Windows
  - Screen Reader: NVDA
  - Language: Angular

**Comment by aarongranick-okta** [comment]
@ramjiseetharaman We have made many accessibility improvements to the widget since version 3.8. The current version is 5.6. Are you able to upgrade to the latest version of the widget to see if these accessibility issues are fixed in the current version of the widget?
